### PR TITLE
feat!: upgrade to Angular 15

### PR DESCRIPTION
**Pull request by aparzi**
The library now supports angular 15.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "angularx-flatpickr",
-  "version": "7.3.0",
+  "version": "8.0.0",
```
Comment (mattlewis92) [inline package.json:3]: Please can you revert this, as it will be bumped automatically on release

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   "description": "An angular 14.0+ wrapper for flatpickr",
```
Comment (mattlewis92) [inline package.json:4]: Can you find and replace all references to `angular 14` -> `angular 15` in the repo?

File: package.json [hunk 3]
```diff
@@ -85,15 +85,13 @@
...
     "tslib": "^2.3.0",
-    "typescript": "~4.7.2",
+    "typescript": "~4.9.5",
     "zone.js": "~0.11.4"
   },
   "dependencies": {
     "tslib": "^2.3.0"
   },
   "peerDependencies": {
-    "@angular/core": ">=5.0.0",
```
Comment (mattlewis92) [inline package.json:95]: Please re-add these peer dependencies and set them to `>=15.0.0` as they can be quick helpful to users

**Review by mattlewis92 — CHANGES_REQUESTED** [review]
Lookin good, thanks! Just had a few minor things. Also you said on your email that you wanted to upgrade to a higher angular version than v15, do you want to do that on this PR as well?

**Comment by aparzi** [comment]
> Looking good, thanks! Just had a few minor things. Also you said on your email that you wanted to upgrade to a higher angular version than v15, do you want to do that on this PR as well?

Hi @mattlewis92,
yes of course but I wanted to do more releases so as to support all versions, not make the direct jump from 14 to 18. Release all versions up to 18 and from 17 also have SSR management?

**Comment by aparzi** [comment]
Hi @mattlewis92,
I have update this PR. Can you check please?

**Review by mattlewis92 — APPROVED** [review]
LGTM! I'm putting this in to a `next` branch for now and then we can release all the major version bumps in a single release

**Pull request merged**